packaging/examples/kafka-access-with-user.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: access.strimzi.io/v1alpha1
```

Comment:
If this is an example can we use `#` comments to explain how it works?

Reply:
I've added a single comment at the top that explains how the config is used.